### PR TITLE
Finished the project Caroline Byström

### PR DESCRIPTION
**Pull request by Frivridna**
Finished the project and aimed at the blue stretch goal. 

I've had som problems with the images in the section "featured projects". Have not found a perfect solution, but a solution. Any tips are appreciated!

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+          </a>
+          <div class="technologies" tabindex="0" aria-label="technologies">
+            <span class="yellow">JavaScript</span>
+            <span class="yellow">NPM</span>
+            <span class="yellow">Node.js</span>
+          </div>
+        </div>
+    </section>
+
+    <section class="my-thoughts">
```
Comment (KarolCislowski) [inline code/index.html]: You can reuse some of the structure of the project cards to create thoughts.

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        </div>
+    </section>
+
+    <section class="featured-projects">
+      <div class="header-featured-projects">
+        <h2 class="header-two">FEATURED PROJECTS</h2>
+      </div>
+      <div class="featured-projects-one">
+        <div class="project-container">
+          <div class="image-container"><a href="https://heuristic-banach-b932cd.netlify.app/">
```
Comment (KarolCislowski) [inline code/index.html]: a whole card should be the link to a hosted project, not only the photo.

Your hover effect makes the rest of the content move. you need to manipulate your paddings and margins to reduce this effect.

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+        <img class="icon-8" src="./assets/img/stackoverflow-logo.svg" alt="Icon and clickable link to Stack overflow">
+      </a>
+    </div>
+    <div class="heading">
+      <strong class="bold-position">Portfolio: Caroline Byström</strong>
+      <h1 class="profession">frontend</h1>
+      <h1 class="profession">developer</h1>
+      <p class="former-job-title">+ choreographer</p>
+    </div>
+      <img class="caroline-image" alt="A headshot image of Caroline Byström" src="./assets/img/IMG_5602.JPG" />
```
Comment (KarolCislowski) [inline code/index.html:29]: The image needs to be slightly moved down to break the Hero section bottom line.

File: code/index.html [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  </header>
+  
+  <main class="main-container">
+    <section class="summary">
```
Comment (KarolCislowski) [inline code/index.html:33]: It would be nice if you create some background differences between 2 consecutive sections to make a clear division.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Your page looks very nice, but it is a little bit too far from the design provided to you by us. 
Please fix the issues I pointed in the comments and it will be a very nice portfolio to show to the recruiters.

File: code/index.html (hunk 2 above)
Comment (Frivridna) [reply code/index.html]: Thank you for the comment. I've fixed this!

**Review by KarolCislowski — APPROVED** [review]
Nice project 😸

**Pull request closed without merging**